src/app/components/About.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const About = () => {
+  return (
+    <section className='flex flex-col items-center gap-4 mb-4'>
```

Comment:
Can you use double quotes here inside the HTML section as per the https://airbnb.io/javascript/react/#quotes